R/controls.R

```diff
@@ -18,32 +181,218 @@ get_model_controls <- function() {
     survey_art_coverage = list(
       name = "survey_art_coverage",
       type = "multiselect",
-      required = TRUE
+      required = FALSE
+    ),
+    include_art_t1 = list(
+      name = "include_art_t1",
+      label = t_("OPTIONS_T1_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T1_HELP"),
+      required = TRUE,
+      options = yes_no_options
+    ),
+    include_art_t2 = list(
+      name = "include_art_t2",
+      label = t_("OPTIONS_T2_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T2_HELP"),
+      required = TRUE,
+      options = yes_no_options
+    ),
+    artattend = list(
+      name = "artattend",
+      type = "select",
+      help_text = t_("OPTIONS_ART_NEIGHBOURING_DISTRICT_HELP"),
+      required = TRUE,
+      options = yes_no_options,
+      value = "true"
+    ),
+    artattend_t2 = list(
+      name = "artattend_t2",
+      type = "select",
+      help_text = t_("OPTIONS_ART_TIME_VARYING_ART_ATTEND_HELP"),
+      required = TRUE,
+      options = yes_no_options,
+      value = "true"
+    ),
+    anc_clients_year2 = list(
+      name = "anc_clients_year2",
+      label = t_("OPTIONS_T2_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T2_HELP"),
+      required = FALSE
+    ),
+    anc_clients_year2_num_months = list(
+      name = "anc_clients_year2_num_months",
+      label = t_("OPTIONS_T2_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_ANC_MONTHS_HELP"),
+      required = TRUE,
+      options = month_options,
+      value = "12"
+    ),
+    anc_prevalence_year1 = list(
+      name = "anc_prevalence_year1",
+      label = t_("OPTIONS_T1_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T1_HELP"),
+      required = FALSE
+    ),
+    anc_prevalence_year2 = list(
+      name = "anc_prevalence_year2",
+      label = t_("OPTIONS_T2_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T2_HELP"),
+      required = FALSE
+    ),
+    anc_art_coverage_year1 = list(
+      name = "anc_art_coverage_year1",
+      label = t_("OPTIONS_T1_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T1_HELP"),
+      required = FALSE
+    ),
+    anc_art_coverage_year2 = list(
+      name = "anc_art_coverage_year2",
+      label = t_("OPTIONS_T2_LABEL"),
+      type = "select",
+      help_text = t_("OPTIONS_T2_HELP"),
+      required = FALSE
+    ),
+    spectrum_population_calibration = list(
```

Comment:
Looking at these, I wonder if you might make an internal function that checks the entries here for misspellings? Something like `control`?

Reply:
Yeah, this sounds like a good idea. Started making that change but it is getting bigger, will add this as another PR